cogs/battling.py

```diff
@@ -44,10 +44,12 @@ def selected(self):
         return self.pokemon[self.selected_idx]
 
     async def send_selection(self):
-        embed = discord.Embed()
-        embed.color = 0xF44336
+        embed = discord.Embed(color=self.bot.embed_color)
```

Comment:
could we possibly subclass discord.Embed to set this automatically?

Reply:
> yeah, that's what I'm doing rn lol